src/globals.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
   // CUSTOM REPLIES
   REPLIES: {
     // GENERAL
+    MOHIT_MKL: "chup mkl",
```

Comment:
we can pick other alternatives here, which does not use offensive language.

For eg: 
1. `tu jaa yaha se, ye scheme tere liye nhi hai`
2. `chal na`